packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/utils/FlowableTests.kt

```diff
@@ -49,6 +49,24 @@ interface FlowableTests {
     @Test
     fun closingRealmDoesNotCancelFlows()
 
+    @Test
+    fun keyPath_topLevelProperty()
```

Comment:
What is the difference between `topLevelProperty` and `nestedProperty`? Aren't they conceptually the same?

Reply:
See https://github.com/realm/realm-kotlin/pull/1547#discussion_r1400692045